src/python/grpcio_observability/grpc_observability/_cyobservability.pyx

```diff
@@ -0,0 +1,336 @@
+# Copyright 2023 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cimport cpython
+from cython.operator cimport dereference
+
+import enum
+import functools
+import logging
+import os
+from threading import Thread
+from typing import List, Mapping, Tuple, Union
+
+import grpc_observability
+
+# Time we wait for batch exporting census data
+# TODO(xuanwn): change interval to a more appropriate number
+CENSUS_EXPORT_BATCH_INTERVAL = float(os.environ.get('GRPC_PYTHON_CENSUS_EXPORT_BATCH_INTERVAL', 0.5))
```

Comment:
Please include a time unit in the name of this env var. I'm assuming seconds, so `GRPC_PYTHON_CENSUS_EXPORT_BATCH_INTERVAL_SECS`